src/components/learner-credit-management/data/utils.js

```diff
@@ -688,8 +688,20 @@ export const startAndEnrollBySortLogic = (prev, next) => {
  * @param isLateRedemptionAllowed
  * @returns {*}
  */
-export const getAssignableCourseRuns = ({ courseRuns, subsidyExpirationDatetime, isLateRedemptionAllowed }) => {
-  const clonedCourseRuns = courseRuns.map(courseRun => ({
+export const getAssignableCourseRuns = ({
+  courseRuns, subsidyExpirationDatetime,
+  isLateRedemptionAllowed,
+  catalogContainsRestrictedRunsData,
+}) => {
+  // First, whittle down the runs to just the ones that are either completely
+  // unrestricted, or restricted but allowed for the current subsidy.
+  const unrestrictedCourseRuns = courseRuns.filter(courseRun => {
```

Comment:
Yea, I constantly face this naming issue.  I hesitated to use "allowed" in this case because the purpose of the surrounding function is to determine which runs are "assignable" which could be conflated with "allowed".  Open to suggestions, or if you think "allowed" works for sufficient code comments.

Reply:
Made irrelevant after refactor that removed the variable declaration entirely.